src/main/java/org/opentripplanner/transit/model/basic/Money.java

```diff
@@ -140,6 +140,11 @@ public Money plus(Money other) {
     return op(other, o -> new Money(currency, amount + o.amount));
   }
 
+  /**
+   * Returns half this instance's amount
```

Comment:
Please document what happens when the amount is an odd number, ie. 99 cent. Also write some tests to verify this behaviour.